tfsdk/schema.go

```diff
@@ -129,6 +129,23 @@ func (s Schema) AttributeAtPath(path *tftypes.AttributePath) (Attribute, error)
 	return a, nil
 }
 
+// Validate performs all Attribute validation.
+func (s Schema) Validate(ctx context.Context, req ValidateSchemaRequest, resp *ValidateSchemaResponse) {
```

Comment:
Does this _need_ to be exported?

Reply:
Likely not -- updated in https://github.com/hashicorp/terraform-plugin-framework/pull/75/commits/3371b5470a35cefa1cb183303c88fc07a8557ef4